linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_create/rule.yml

```diff
@@ -25,9 +25,14 @@ severity: medium
 
 identifiers:
     cce@rhel7: CCE-86115-3
+    cce@rhel8: CCE-88435-3
+    cce@rhel9: CCE-88436-1
 
 references:
     cis@alinux3: 4.1.3.26
+    cis@rhel7: 4.1.16
```

Comment:
Correct. It was included by mistake there. The changes in this commit are only for RHEL 8 and RHEL 9. I will amend the respective commit to avoid confusion.